docs/03-design/LZ-03-03.adoc

```diff
@@ -5,7 +5,7 @@
 Softwarearchitekt:innen können grundlegende Vorgehensweisen der Architekturentwicklung benennen, erklären und anwenden, beispielsweise:
 
 * Top-down- und Bottom-up-Vorgehen beim Entwurf (R1)
-* Sichtenbasierte Architekturentwicklung (R1)
+* Sichtenbasierte Architekturentwicklung, siehe <<rozanskiwoods>>, <<kruchten95>> (R1)
```

Comment:
You added 3 book references but the 3rd is missing here: starkeetalarc42

Do we need all the books referenced from somewhere in the learning goals?

Reply:
> Do we need all the books referenced from somewhere in the learning goals?

IMHO that's the right place instead of "just adding a bunch of books at the end". Our problem here is that we DO NOT (and should not) provide a concise explanation of the concepts to be learned in the curriculum but should point to them (either glossary or literature).